doc/Security.xml

```diff
@@ -1063,6 +1065,34 @@
               secp256r1))</programlisting>
       <para>The evaluated signature is further appended to the encoded header and payload with a '.' separator, to get the final JWT. Appendix <xref linkend="_Ref460769599"/> demonstrates the construction or a JWT.</para>
       <para>The signature is used to protect the JWT data integrity and JWT source authenticity.</para>
+      </section>
+      <section>
+        <title>Configuration</title>
+        <para>The folowing parameters are defined:</para>
+        <variablelist>
+          <varlistentry>
+            <term>Audiences</term>
+            <listitem><para>One or more audience strings. A device shall reject a token if none of the provided strings matches.</para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>TrustedIssuer</term>
+            <listitem><para>Optional list of trusted issuer metaadata URIs. A device shall reject a token if the list is
```

Comment:
typo change: metaadata URIs->metadata URIs

Reply:
Thanks for thorough reading.